### PR TITLE
(IRO-2638) Added hash Icon to transaction table.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
https://linear.app/ironfish/issue/IRO-2643/new-requests-feedback-on-pr-httpsgithubcomiron-fishblock

**Comment by azhurauski-sc** [comment]
@skylarrichard123 please check badge positioning, table header will be fixed a bit later.

**Comment by ndemidovich-sc** [comment]
@azhurauski-sc Please fix build.

File: components/TransactionsTable/TransactionsTable.tsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ const TAG_COLUMN: ColumnProps<TransactionType> = {
     transaction?.spends.length === 0 && (
       <InfoBadge
         mt={{ base: '1rem', lg: 0 }}
+        mx={{ base: '32px', sm1: 0 }}
```
Comment (ndemidovich-sc) [inline components/TransactionsTable/TransactionsTable.tsx]:
```suggestion
        mx={{ base: '2rem', sm1: 0 }}
```

**Review by ndemidovich-sc — APPROVED** [review]

**Review by skylarrichard123 — APPROVED** [review]

**Pull request merged**